install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -145,8 +163,8 @@ spec:
                 at the desired state
               type: integer
               format: int32
-  version: v1alpha1
+  version: v1beta1
   versions:
-  - name: v1alpha1
+  - name: v1beta1
```

Comment:
@hardys @stbenjam are we advancing to version v1beta1 with this change? Or do we continue to be in version v1alpha1?

Reply:
I'm not sure it matters as it's effectively an internal API between installer and MAO - we'll need to ensure things work on upgrade with an old CR version though - @dhellmann any thoughts?

Reply:
Changing the version number will mean existing resources will appear to not be there, unless we also provide conversion webhooks to support fetching them. That's going to introduce extra complications in the upgrade/downgrade process. Let's get the new operator in place using the existing API version without any changes and then consider whether we are ready to call it beta or v1.